src/main/java/appeng/items/contents/PortableCellViewer.java

```diff
@@ -83,14 +86,12 @@ public double extractAEPower( double amt, final Actionable mode, final PowerMult
 	}
 
 	@Override
-	public IMEMonitor<IAEItemStack> getItemInventory()
-	{
-		return this;
-	}
-
-	@Override
-	public IMEMonitor<IAEFluidStack> getFluidInventory()
+	public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEMonitor<T> getInventory( IStorageChannel<T> channel )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') C is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS2326)